examples/custom-action/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Having a package-lock would be better.

We don't have one for the other examples but I think that it's a mistake.